### PR TITLE
Feature/retry+remove jobs

### PR DESCRIPTION
**Pull request by SimonErm**
Adresses #44

### CODE REVIEW

File: example/App.tsx [hunk 1]
```diff
@@ -56,11 +56,34 @@ export default class App extends React.Component<IAppProps, IAppState> {
...
+                    onPress={async () => {
+                        let jobs = await queue.getJobs();
+                        let jobRequeues = jobs.filter(job => job.failed !== '').map(async job => await queue.requeueJob(job))
+                        await Promise.all(jobRequeues)
+                    }}
+                />
+                <Button
+                    title='getJobs'
+                    onPress={async () => {
+                        let jobs = await queue.getJobs();
```
Comment (myckhel) [inline example/App.tsx:85]: Seeing this in a long run.
What if i have 1000s of jobs in store.
Would it be a good idea to retrieve all sometimes?

Maybe we could add options to limit the records.

Comment (SimonErm) [reply example/App.tsx:85]: Yes i think this is a good idea, but this should be addressed in an another PR

**Comment by myckhel** [comment]
you forgot retry method?

**Comment by SimonErm** [comment]
No i just had a bad PR naming. Instead of a retry method there is a requeue method now.

**Comment by myckhel** [comment]
Ohkay.

**Pull request merged**